src/cltk/alphabet/lat.py

```diff
@@ -1,12 +1,471 @@
+    that may creep into the text. Caution using this elsewhere.
+    :param text:
+    :return:
+
+    >>> dehyphenate('quid re-tundo hier')
+    'quid retundo hier'
+    """
+
+    idx_to_omit = []
+    for item in HYPHENATED.finditer(text):
+        idx_to_omit.insert(0, item.span())
+    for start, end in idx_to_omit:
+        text = text[:start] + text[start:end].replace("-", "") + text[end:]
+    return text
+
+
+def swallow(text: str, pattern_matcher: re.Pattern) -> str:
+    """
+    Utility function internal to this module
+
+    :param text: text to clean
+    :param pattern_matcher: pattern to match
+    :return: the text without the matched pattern; spaces are not substituted
+    """
+    idx_to_omit = []
+    for item in pattern_matcher.finditer(text):
+        idx_to_omit.insert(0, item.span())
+    for start, end in idx_to_omit:
+        text = text[:start] + text[end:]
+    return text.strip()
+
+
+def swallow_braces(text: str) -> str:
+    """
+    Remove Text within braces, and drop the braces.
+
+    :param text: Text with braces
+    :return: Text with the braces and any text inside removed
+
+    >>> swallow_braces("{PRO P. QVINCTIO ORATIO} Quae res in civitate {etc}... ")
+    'Quae res in civitate ...'
+
+    """
+    return swallow(text, BRACE_STRIP)
+
+
+def drop_latin_punctuation(text: str) -> str:
+    """
+    Drop all Latin punctuation except the hyphen and obelization markers, replacing the punctuation
+    with a space.
+    Please collapsing hyphenated words and removing obelization marks separately beforehand.
+
+    The hyphen is important in Latin tokenization as the enclitic particle `-ne`
+    is different than the interjection `ne`.
+
+    :param text: Text to clean
+    :return: cleaned text
+
+    >>> drop_latin_punctuation('quid est ueritas?')
+    'quid est ueritas '
+
+    >>> drop_latin_punctuation("vides -ne , quod , planus est ")
+    'vides -ne   quod   planus est '
+
+    >>> drop_latin_punctuation("here is some trash, punct \/':;,!\?\._『@#\$%^&\*okay").replace("  ", " ")
+    'here is some trash punct okay'
+
+    """
+    text = NUMERALS.sub(" ", text)
+    text = LATIN_PUNCT.sub(" ", text)
+    text = QUOTES.sub(" ", text)
+    return text
+
+
+def remove_accents(text: str) -> str:  # pylint: disable=too-many-statements
+    """
+    Remove accents; note: AE replacement and macron replacement should happen elsewhere, if desired.
+    :param text: text with undesired accents
+    :return: clean text
+
+    >>> remove_accents('suspensám')
+    'suspensam'
+
+    >>> remove_accents('quăm')
+    'quam'
+
+    >>> remove_accents('aegérrume')
+    'aegerrume'
+
+    >>> remove_accents('ĭndignu')
+    'indignu'
+
+    >>> remove_accents('îs')
+    'is'
+
+    >>> remove_accents('óccidentem')
+    'occidentem'
+
+    >>> remove_accents('frúges')
+    'fruges'
 
-def normalize_lat(text: str) -> str:
+    """
+    text = text.replace(r"á", "a")  # suspensám
```

Comment:
Am totally good with this. Is amazing it takes so much work to spell these out!

Is the `r` required here?

Reply:
Surprisingly yes in a few cases, so I adopted it to be consistent. It's ugly, but the rogue accents are uglier.
Btw, none of the transformations are just my fancy, they came from working with our regular corpora stores: Phi5, tesserae, latin library, etc.